From 46a49ef56b470d839b766f4ff066cc2976c9e7e9 Mon Sep 17 00:00:00 2001
From: Sohom <sohomdatta1+git@gmail.com>
Date: Mon, 6 Mar 2023 10:37:25 -0500
Subject: [PATCH] Change SSL mode to disabled

---
 post-processor/main.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/post-processor/main.go b/post-processor/main.go
index 68d8fa8..897650d 100644
--- a/post-processor/main.go
+++ b/post-processor/main.go
@@ -403,7 +403,7 @@ func invoke(args []string, topLevel bool) error {
 			if outputFormat == "mongresql" {
 				// Do we need to connect to PG? (we rely on the PGxxx environment variables being set...)
 				if aggCtx.SQLDb == nil {
-					aggCtx.SQLDb, err = sql.Open("postgres", "")
+					aggCtx.SQLDb, err = sql.Open("postgres", "sslmode=disable")
 					if err != nil {
 						return err
 					}
-- 
2.39.2

